cachet-tools/Dockerfile

```diff
@@ -1,21 +1,34 @@
 # Get centos base image
 # Use FROM scratch for blank image
-FROM python:2
+FROM docker.io/centos/python-36-centos7
 
 # who are you
-MAINTAINER bennyturns
+MAINTAINER aarif
 
-COPY requirements.txt /
+# copy file into place
+COPY update_cachet.py /
+COPY services.py /
+COPY hosted-services.json / 
+COPY populate-cachet.py /
+COPY purge-cachet.py /
+COPY requirements.txt / 
+CMD ["ls -l /"]
 
 # Run when creating image
 RUN pip install --no-cache-dir -r /requirements.txt
 
-# copy file into place
-COPY cachet_update_daemon.py / 
+# Arif's project
```

Comment:
do these `ENV` lines imply you have to have those env vars set for this docker build to work?

Reply:
These will be set via the deployment config when running in openshift.  And when running locally, perhaps we pass the env vars in when the container is started?

Reply:
yes.  At the moment this is how I could make it work.  Once the "oc new-app ..." has created the deployment configuration, I manually create env vars and assign values.  

When using APPSRE, I may pass in args to new-app or create a secret.